### ISSUE TITLE
correction random.vitturi.fsb

### ISSUE THREAD

**Issue by mattsteeldue** [programandala-net/afera#2]
\ I recognize this is a piece of code I wrote as experiment, I wish to suggest the following correction:
\ This word accepts n1 < 16382 and gives a pseudo-random number n2 between 0 and n1.
\ It uses two primes to produce a unique sequence that repeats itself after 16381 iterations.
: RND ( n1 -- n2 )
  1+ 8191 SYS-SEED @ U* 1. D+
  16381 U/ DROP DUP SYS-SEED !
  SWAP MOD ;

**Comment by programandala-net** [comment]
Yes, the `rnd` from your vForth 1.3 is one of the 3 variants included in the library, as credited in the source:
<https://github.com/programandala-net/afera/blob/master/src/random.vitturi.fsb>. I will update it with your improved version and your explanation. Thank you.

Anyway, in [Tron 0xF](https://github.com/programandala-net/tron-0xf), the only project I wrote using this fig-Forth library, I used the variant by Leo Brodie: 
<https://github.com/programandala-net/afera/blob/master/src/random.fsb>. I don't remember the reason.
